flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionCapacitySchedulerITCase.java

```diff
@@ -503,7 +503,7 @@ private void testDetachedPerJobYarnClusterInternal(String job) throws Exception
 				"-yjm", "768m",
 				// test if the cutoff is passed correctly (only useful when larger than the value
 				// of containerized.heap-cutoff-min (default: 600MB)
-				"-yD", "yarn.heap-cutoff-ratio=0.7",
+				"-yD", "containerized.heap-cutoff-ratio=0.7",
```

Comment:
can we derive the key from an actual `ConfigOption` instead of hard-coding it?

Reply:
Sounds reasonable. Updated.